MonoGame.Framework/Point.cs

```diff
@@ -77,6 +77,37 @@ public Point(int x, int y)
             this.Y = y;
         }
 
+        /// <summary>
+        /// Creates a new instance of <see cref="Point"/> struct, with the specified position.
+        /// </summary>
+        /// <param name="value">The x and y coordinates in 2d-space.</param>
+        public Point(int value)
+        {
+            this.X = value;
+            this.Y = value;
+        }
+
+        /// <summary>
+        /// Creates a new instance of <see cref="Point"/> struct, with the specified position.
+        /// </summary>
+        /// <param name="x">The x coordinate in 2d-space.</param>
+        /// <param name="y">The y coordinate in 2d-space.</param>
+        public Point(float x, float y)
```

Comment:
The method itself is not necessary, I agree - because you can always write your own extension method. But for constructors it is the only way to decrease amount of space needed to creating instance of class/struct.

> Though wow Vector2 does indeed have such a useless constructor

It is exist for specific reasons. For example - you can create a surface with same width and height with this overload in one click.